salt/modules/boto_elasticache.py

```diff
@@ -84,7 +84,121 @@ def exists(name, region=None, key=None, keyid=None, profile=None):
+
+    CLI example::
+
+        salt myminion boto_elasticache.create_replication_group myelasticache myprimarycluster description
+    '''
+    conn = _get_conn(region, key, keyid, profile)
+    if not conn:
+        return None
+    try:
+        cc = conn.create_replication_group(name, primary_cluster_id,
+                                           replication_group_description)
+        if not wait:
+            log.info('Created cache cluster {0}.'.format(name))
+            return True
+        while True:
+            time.sleep(3)
+            config = describe_replication_group(name, region, key, keyid, profile)
+            if not config:
+                return True
+            if config['status'] == 'available':
+                return True
+    except boto.exception.BotoServerError as e:
+        msg = 'Failed to create replication group {0}.'.format(name)
+        log.error(msg)
+        log.debug(e)
+        return {}
+
+
+def describe_replication_group(name, region=None, key=None, keyid=None,
+                               profile=None, parameter=None):
+    '''
+    Get replication group information.
+
+    CLI example::
+
+        salt myminion boto_elasticache.describe_replication_group mygroup
+    '''
+    conn = _get_conn(region, key, keyid, profile)
+    if not conn:
+        return None
+    try:
+        cc = conn.describe_replication_groups(name)
+    except boto.exception.BotoServerError as e:
+        msg = 'Failed to get config for cache cluster {0}.'.format(name)
+        log.error(msg)
+        log.debug(e)
+        return {}
+    ret = odict.OrderedDict()
+    cc = cc['DescribeReplicationGroupsResponse']['DescribeReplicationGroupsResult']
+    cc = cc['ReplicationGroups'][0]
+
+    attrs = ['status', 'description', 'primary_endpoint',
+             'member_clusters', 'replication_group_id',
+             'pending_modified_values', 'primary_cluster_id',
+             'node_groups']
+    for key, val in six.iteritems(cc):
+        _key = boto.utils.pythonize_name(key)
+        if _key == 'status':
+            if val:
+                ret[_key] = val
+            else:
+                ret[_key] = None
+        if _key == 'description':
+            if val:
+                ret[_key] = val
+            else:
+                ret[_key] = None
+        if _key == 'replication_group_id':
+            if val:
+                ret[_key] = val
+            else:
+                ret[_key] = None
+        if _key == 'member_clusters':
+            if val:
+                ret[_key] = val
+            else:
+                ret[_key] = None
+        if _key == 'node_groups':
+            if val:
+                ret[_key] = val
+            else:
+                ret[_key] = None
+        if _key == 'pending_modified_values':
+            if val:
+                ret[_key] = val
+            else:
+                ret[_key] = None
+    return ret
+
+
+def get_config(name, region=None, key=None, keyid=None, profile=None):
```

Comment:
Did this argspec change? We don't want to break existing calls to this function.